core/drivers/crypto/se050/core/sub.mk

```diff
@@ -7,7 +7,7 @@ srcs-y += storage.c
 srcs-$(CFG_NXP_SE05X_RSA_DRV) += rsa.c
 srcs-$(CFG_NXP_SE05X_ECC_DRV) += ecc.c
 srcs-$(CFG_NXP_SE05X_CTR_DRV) += ctr.c
-srcs-$(CFG_NXP_SE05X_HUK_DRV) += huk.c
+srcs-$(CFG_NXP_SE05X_DIEID_DRV) += die_id.c
```

Comment:
1. Minor: `DIE_ID` would be a bit easier to read. Your choice.
2. Should a few lines be changed in `die_id.c`? (`se050_huk`, etc.)